lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -125,10 +122,7 @@ export default function LMSGrader({
             />
           </li>
           <li className="LMSGrader__student-grade">
-            <SubmitGradeForm
-              student={getCurrentStudent()}
-              disabled={currentStudentIndex < 0}
-            />
+            <SubmitGradeForm student={getCurrentStudent()} />
```

Comment:
Awe, okay

we may need to change the state variable (line 36) from 

`const [currentStudentIndex, setCurrentStudentIndex] = useState(-1);`

to 

`const [currentStudentIndex, setCurrentStudentIndex] = useState(null);`

The negative index may be a bit misleading now since it has no purpose.

Reply:
I tried making this change but it turns out that several other places in this component and also `StudentSelector` make use of the fact that a negative `currentStudentIndex` is used to represent "no student selected". For the moment I have simply documented this fact in the `StudentSelector` props.

Using `null` would have the advantage that TS would be able to offer more help in terms of ensuring that you checked for it before indexing into the students list.